### PR TITLE
Bump ratatool to 0.4.9

### PR DESCRIPTION
**Pull request by leschaller1**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 helios*             @spotify/helios
-ratatool.rb         @spotify/flatmap @spotify/blizzard
+ratatool.rb         @spotify/flatmap @spotify/dmx
```
Comment (leschaller1) [inline .github/CODEOWNERS:2]: created here https://github.com/orgs/spotify/teams/dmx/members

**Review by benkonz — COMMENTED** [review]
hmm, I get a different checksum on my laptop

```
<~/Downloads>-> shasum -a 256 ratatool-cli-0.4.9.tar.gz
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  ratatool-cli-0.4.9.tar.gz
```

**Comment by benkonz** [comment]
> hmm, I get a different checksum on my laptop
> 
> ```
> <~/Downloads>-> shasum -a 256 ratatool-cli-0.4.9.tar.gz
> e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  ratatool-cli-0.4.9.tar.gz
> ```

along with `curl https://github.com/spotify/ratatool/releases/download/v0.4.9/ratatool-cli-0.4.9.tar.gz --output ~/Downloads/ratatool-cli-0.4.9.tar.gz`

**Comment by leschaller1** [comment]
@benkonz Huh weird yeah when I download the one I uploaded I get your output...but when I run it on the file that I used to upload I still get my original output

**Comment by leschaller1** [comment]
```
shasum -a 256 ratatool-cli/target/ratatool-cli-0.4.9.tar.gz         
0bf47798a398977915edf21fc65948a4c6b848de41c00af9986efd5669bba111  ratatool-cli/target/ratatool-cli-0.4.9.tar.gz
```

```
shasum -a 256 ~/Downloads/ratatool-cli-0.4.9.tar.gz  
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  /Users/laurenschaller/Downloads/ratatool-cli-0.4.9.tar.gz
```

**Comment by benkonz** [comment]
@leschaller1 working now! I think my curl command was off or something, but when I downloaded the file through the github UI, I got the same sha!

**Review by benkonz — APPROVED** [review]

**Comment by leschaller1** [comment]
@benkonz I think it was an issue on my end, I reuploaded it (the old one was also giving me the same issue you were seeing). All good now!

**Pull request merged**